Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -811,6 +865,47 @@ def check_font_has_a_license(fb, licenses):
 
 @register_test
 @old_style_test(
+    id='com.google.fonts/test/029'
+  , conditions=['license']
+  , priority=CRITICAL
+)
+def check_copyright_entries_match_license(fb, ttFont, license):
+  """Check copyright namerecords match license file"""
+  failed = False
+  for license in ['OFL.txt', 'LICENSE.txt']:
+    if license not in ttFont.reader.file.name:
```

Comment:
And I really don't get what this is supposed to do. Just tried this:

```
>>> ttfont = TTFont('./RobotoSlab-Regular.ttf')
>>> ttfont.reader.file.name
 './RobotoSlab-Regular.ttf'
```

So how is `"OFL.txt" or "LICENSE.txt" supposed to be in the file name?